lexicons/app/bsky/feed/detach.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.detach",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining post URIs detached from a root post. The record key (rkey) of the detach record must match the record key of the root post in question, and that record must be in the same repository.",
```

Comment:
"detaching" works against both replies and quote-posts, right? either way we should probably clarify here in the description was "detatch" actually means.